packages/common/src/block.ts

```diff
@@ -0,0 +1,19 @@
+import { Maybe, Types } from '@dotstats/common';
+
+export function blockAverage(blockTimes: Array<number>): Maybe<Types.Milliseconds> {
```

Comment:
You never return a nullish value, so `Maybe` around return type isn't necessary.